smart-contracts/wasm-chain-integration/src/utils.rs

```diff
@@ -17,39 +18,6 @@ use concordium_wasm::{
 use rand::{prelude::*, RngCore};
 use std::{collections::BTreeMap, default::Default};
 
-#[derive(Debug, Clone, Copy)]
-pub enum WasmVersion {
-    V0,
-    V1,
-}
-
-impl std::str::FromStr for WasmVersion {
-    type Err = anyhow::Error;
-
-    fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s {
-            "V0" | "v0" => Ok(WasmVersion::V0),
-            "V1" | "v1" => Ok(WasmVersion::V1),
-            _ => anyhow::bail!("Unsupported version: '{}'. Only 'V0' and 'V1' are supported.", s),
-        }
-    }
-}
-
-impl WasmVersion {
-    /// Get the version from the cursor. This is not a Serial implementation
-    /// since it uses big-endian.
-    pub fn read(source: &mut std::io::Cursor<&[u8]>) -> anyhow::Result<WasmVersion> {
```

Comment:
Is this not needed anywhere? Cause I don't see it moved.

Reply:
The place where I found it used was in `cargo-concordium`, but there we have access to `Deserial::deserial` (the BE one) which behaves the same, so I removed it.

Reply:
Ok.